config/opensearch_dashboards.yml

```diff
@@ -313,3 +313,7 @@
 
 # Set the value to true to enable workspace feature
 # workspace.enabled: false
+
+# Set the value to true to enable Ui Metric Collectors in Usage Collector
```

Comment:
Let's call our this might cause permission issue to OpenSearch. turn on with caution

Reply:
Updated the comments